streaming/README.md

```diff
@@ -488,6 +488,12 @@ You can monitor the replication process in two ways:
 1. **Netdata Monitoring**: access the Netdata Monitoring section and look for the Replication charts. 
 2. **Streaming Function**: use the Streaming function (Top) to see the replication status of children nodes. This function provides real-time insights into the replication status of each child node.
 
+### Replication and storage
+
+When netdata replicates data from child to parent, the agent will replicate data stored in [tier 0](https://learn.netdata.cloud/docs/configuring/optimizing-metrics-database/change-how-long-netdata-stores-metrics#effect-of-storage-tiers-and-disk-space-on-retention),
+because other tiers are calculated from its data. The default storage values allow `tier 0` to have `12 days` of retention.
```

Comment:
Where did you get 12 days from?

Reply:
It was a talk with @stelfrag in slack.

Reply:
I'm not sure if there is a default limit, it very much depends on the number of metrics being collected, which can vary greatly.

Reply:
That calculation was based on specific data (expected retention from /node_instances)

Reply:
Then we just need to say that if the child instance's database mode is dbengine, it replicates only tier 0. The other levels are not replicated, but are calculated by the parent instance using tier 0. Regarding retention, they should check the expected retention time from /node_instances rather than some random number of days.

Reply:
I will push a commit updating, thanks for the suggestion.